spacemesh/v2alpha1/tx.proto

```diff
@@ -22,10 +22,13 @@ message Transaction {
     TRANSACTION_TYPE_UNSPECIFIED = 0;
     TRANSACTION_TYPE_SINGLE_SIG_SEND = 1;
     TRANSACTION_TYPE_SINGLE_SIG_SPAWN = 2;
-    TRANSACTION_TYPE_SINGLE_SIG_SELFSPAWN = 3;
+    TRANSACTION_TYPE_SINGLE_SIG_SELFSPAWN = 3; // deprecated
```

Comment:
How about using [deprecated option](https://protobuf.dev/programming-guides/proto3/#options)?
```suggestion
    TRANSACTION_TYPE_SINGLE_SIG_SELFSPAWN = 3 [deprecated = true];
```

Reply:
Cool, I didn't know that